src/experiments.js

```diff
@@ -47,6 +51,102 @@ export function isCanary(win) {
 }
 
 
+function getExperimentsKey_() {
+  let key;
+  const url = '' //TODO(kmh287): fill this in
```

Comment:
Not sure how to set up a key to download. Is the extra request worth the time it would take to perform another request?